BookingProject/BookingWebAPI/BookingWebApi.Application/Models/ApiResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.Models
+{
+    public class ApiResponse<T>
```

Comment:
Looks like this belongs to Presentation layer, but not Application. For application layer you can use `Result<T>` pattern and then map it to your `ApiResponse<T>`